### ISSUE TITLE
Block quotes green text

### ISSUE THREAD

**Issue by jholdstock** [decred/dcrdocs#759]
As a result of #566, the block quote text was changed to green.

At the time, blockquote was mostly being used for headers and so the decision made some sense. I think this feels like a bit of a mistake in hindsight as it makes the blockquotes quite ugly when used for their actual purpose.

We should replace the current bq headers with an actual header element (I dont think we use ##### anywhere), and change the block quote style so it can be used for larger chunks of text.

Make block quote great again.

**Comment by jholdstock** [comment]
The PR above replaces the existing usage of blockquotes with headers. Once this is merged I will create a new PR to use blockquotes on the politeia sample proposals page
